app/js/editTests.js

```diff
@@ -0,0 +1,38 @@
+const testId = location.hash.replace('#', '');
+
+function populateQuestions(testId) {
+    populateHandlebars('.tableBody', 'js/templates/questionTable.hbs', `question?test_id=${testId}`).then(() => {
+        document.querySelectorAll(".delete-question").forEach((btn) => {
+            btn.addEventListener('click', async (clickedBtn) => {
+                const {id} = clickedBtn.target.parentElement.dataset;
+                await deleteQuestion(id);
```

Comment:
theres no check to see if this actually worked? your not using the return value. what if the API fails?